satpy/composites/__init__.py

```diff
@@ -743,6 +744,178 @@ def __call__(self, projectables, optional_datasets=None, **info):
         return proj
 
 
+class ParallaxCorrector(CompositeBase):
+    """Parallax Corection
+
+    Ref:
+        code: https://github.com/ORAC-CC/orac/blob/master/post_processing/correct_parallax.F90
+        paper: 10.3390/rs12030365 (doi)
+    """
+
+    def _parallax_corr(self, optional_datasets,
+                       sat_lon, sat_lat, sat_h,
+                       lon, lat):
+        """Applying Parallax Correction"""
+        # varius earth radius information
+        radius_eq = 6378.077
+        radius_pole = 6356.577
```

Comment:
Can we have an option to take these from the area def, if possible? The GEOsats should all define these in their projection description so it'd be better to use those values, then fall back to the hardcoded values above for satellites that do not have the values in their metadata.

Reply:
Here's the projection of AGRI:

```
Projection: {'a': '6378140', 'h': '35786000', 'lon_0': '104.699996948242', 'no_defs': 'None', 'proj': 'geos', 'rf': '298.257223562999', 'type': 'crs', 'units': 'm', 'x_0': '0', 'y_0': '0'}
```

Just have `a` ....

How to deal with this issue